recipes/libv8/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "libv8" %}
+{% set version = "7.5.288.22" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: "https://github.com/v8/v8/archive/{{ version }}.tar.gz"
+    sha256: df12222ebdb9538f62434785ea246f1d27f198b5d674e0577dd1487ffe53b6e2
+  # We need to use git instead of tarballs as tarballs aren't stable and
+  # have a different on every download.
+  # Sychronized hashes from https://github.com/Homebrew/homebrew-core/blob/2a8b9b60c832d25c8b2013d641e49a886ccbe0f7/Formula/v8.rb
+  - git_url: "https://chromium.googlesource.com/chromium/src/build.git"
+    git_rev: a0b2e3b2708bcf81ec00ac1738b586bcc5e04eea
+    folder: build
+  - git_url: "https://chromium.googlesource.com/chromium/src/third_party/jinja2.git"
+    git_rev: b41863e42637544c2941b574c7877d3e1f663e25
+    folder: third_party/jinja2
+  - git_url: "https://chromium.googlesource.com/chromium/src/third_party/markupsafe.git"
+    git_rev: "8f45f5cfa0009d2a70589bcda0349b8cb2b72783"
+    folder: third_party/markupsafe
+  - git_url: "https://chromium.googlesource.com/external/github.com/google/googletest.git"
+    git_rev: "b617b277186e03b1065ac6d43912b1c4147c2982"
+    folder: third_party/googletest/src
+  - git_url: "https://chromium.googlesource.com/chromium/src/base/trace_event/common.git"
+    git_rev: "ebb658ab38d1b23183458ed0430f5b11853a25a3"
+    folder: base/trace_event/common
+  - git_url: "https://chromium.googlesource.com/chromium/deps/icu.git"
+    git_rev: "35f7e139f33f1ddbfdb68b65dda29aff430c3f6f"
```

Comment:
Shouldn't these be conda-forge packages? I think a lot of them are already

Reply:
Yes, they are. But it is not straight-forward to let v8 use them. `icu` is the even  weirder case where need to check out the source and patch it to use conda-forge's icu package.

Reply:
OK, I think you can use a URL like `https://chromium.googlesource.com/chromium/deps/icu.git/+archive/35f7e139f33f1ddbfdb68b65dda29aff430c3f6f.tar.gz` to get the source, though. Rather than the git repo

Reply:
I tried that but we I cannot use checksums on these tarballs as they are not stable.

Reply:
Yep, that is just something that you'll need to updated.  They shouldn't be so unstable as you won't be able to build the package though.  That implies that the tarballs change super frequently

Reply:
No, the tarballs literally have a different sha256 on every download. So they are then actually unpredictable.

Reply:
Wow, I can confirm that.  That is messed up.

Reply:
@scopatz Should I then keep the git URLs or use the archives without a sha?

Reply:
My pref would be the tarballs without a hash, since they are smaller to download.